Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/GraphData/Nodes/MaterialOutputs/StandardPBR/MaterialGraphName_SurfaceEval.azsli

```diff
@@ -91,7 +91,7 @@ Surface EvaluateSurface_MaterialGraphName(
         surface.clearCoat.roughness = max(inClearCoatRoughness, 1e-3);
 
         // Flip normal if back face is rendered   
-        surface.clearCoat.normal = normalize(o_clearCoat_normal_override_enabled ? GetWorldSpaceNormal((float2)inClearCoatNormal, vertexNormal, tangents[0], bitangents[0]) : vertexNormal);
```

Comment:
Here is an example where I think using vertexNormalWS would be beneficial. It's a bit confusing to see That GetWorldSpaceNormal takes a normalWS as input.

Reply:
I see your point. What about renaming the function as well?